spec/features/survey_response_new_survey_spec.rb

```diff
@@ -0,0 +1,74 @@
+require "rails_helper"
+
+feature "Submitting a new code confidence survey", js: true do
```

Comment:
Do these tests need javascript? Just checking.

Reply:
No but was seeing if it made a difference with the issue I was having with the last spec. I can delete it now.